cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ZookeeperCluster.java

```diff
@@ -78,17 +78,20 @@
 
     // Zookeeper configuration
     private TlsSidecar tlsSidecar;
+    private boolean isSnapshotCheckEnabled;
 
     public static final Probe DEFAULT_HEALTHCHECK_OPTIONS = new ProbeBuilder()
             .withTimeoutSeconds(5)
             .withInitialDelaySeconds(15)
             .build();
     private static final boolean DEFAULT_ZOOKEEPER_METRICS_ENABLED = false;
+    private static final boolean DEFAULT_ZOOKEEPER_SNAPSHOT_CHECK_ENABLED = true;
 
     // Zookeeper configuration keys (EnvVariables)
     public static final String ENV_VAR_ZOOKEEPER_NODE_COUNT = "ZOOKEEPER_NODE_COUNT";
     public static final String ENV_VAR_ZOOKEEPER_METRICS_ENABLED = "ZOOKEEPER_METRICS_ENABLED";
     public static final String ENV_VAR_ZOOKEEPER_CONFIGURATION = "ZOOKEEPER_CONFIGURATION";
+    public static final String ENV_VAR_ZOOKEEPER_SNAPSHOT_CHECK_ENABLED = "ZOOKEEPER_SNAPSHOT_CHECK_ENABLED";
```

Comment:
I would name this `STRIMZI_ZOOKEEPER_SNAPSHOT_CHECK_ENABLED` or something similar. All the env vars starting with `ZOOKEEPER...` are badly named because it suggests an actual Zookeeper environment variable. But since we have bunch of other like that I'm fine if you ignore this as well.

Reply:
Makes sense, I have updated the env vars.